API.md

```diff
@@ -254,6 +281,18 @@ Takes a function which accepts a node-style callback and returns a new function
 const upload = promisify(s3.upload, s3)
 ```
 
+### setProp
+
+```haskell
+setProp : String -> { k: v } -> a -> { k: v }
+```
+
+Accepts a key, object and any value.  Sets the property `key` on the object to `value`.
```

Comment:
`an object`? To me, this helps with saying there are three parameters instead of `key, object` being one thing lumped together